frontend/src/text_resources.js

```diff
@@ -550,7 +561,12 @@ let LOCALIZE = new LocalizedStrings({
             "Les principaux objectifs de l’Unité des SC sont d’évaluer de façon continue les programmes de formation offerts par les organisations de la fonction publique, effectuer des vérifications internes et externes pour les partenaires et les clients, et surveiller la diffusion de l’information (p. ex., évaluer le contenu des outils en ligne, rédiger les documents relatifs aux programmes de formation). L’Unité des SC est composée de l’Équipe de l’assurance de la qualité, l’Équipe du service et soutien, l’Équipe des vérifications et de l’Équipe des formations en ligne.",
           orgChart: {
             desciption: "Organigramme (CDO)",
-            link: "FR Image Description (Coming Soon)"
+            link: "FR Image Description"
+          },
+          dialog: {
+            title: "FR The Organizational Chart of the Organizational Development Council’s (ODC)",
```

Comment:
Your screenshots indicate these titles should be shorter or that we need to fix the css to allow for longer titles.

Reply:
Shortened titles in 9f4c88a